nerfstudio/engine/trainer.py

```diff
@@ -75,6 +75,8 @@ class TrainerConfig(ExperimentConfig):
     """Maximum number of iterations to run."""
     mixed_precision: bool = False
     """Whether or not to use mixed precision for training."""
+    use_grad_scaler: bool = False
+    """Use gradient scaler even if amp is disabled"""
```

Comment:
Reword incase people don't know what amp is
"""Use gradient scaler even if mixed_precision is disabled"""